File: sdkwork-backend-sdk-typescript/src/types/page-plus-ai-generation-content-vo.ts
import type { PageableObject } from './pageable-object';
import type { PlusAiGenerationContentVO } from './plus-ai-generation-content-vo';
import type { SortObject } from './sort-object';

export interface PagePlusAiGenerationContentVO {
  totalPages?: number;
  totalElements?: number;
  size?: number;
  content?: PlusAiGenerationContentVO[];
  number?: number;
  numberOfElements?: number;
  first?: boolean;
  last?: boolean;
  sort?: SortObject;
  pageable?: PageableObject;
  empty?: boolean;
}

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PagePlusAiGenerationContentVO.kt
package com.sdkwork.backend

data class PagePlusAiGenerationContentVO(
    val totalElements: Int? = null,
    val totalPages: Int? = null,
    val size: Int? = null,
    val content: List<PlusAiGenerationContentVO>? = null,
    val number: Int? = null,
    val first: Boolean? = null,
    val last: Boolean? = null,
    val numberOfElements: Int? = null,
    val sort: SortObject? = null,
    val pageable: PageableObject? = null,
    val empty: Boolean? = null
)
